### DISCUSSION TITLE
java.lang.ClassNotFoundException: /resource/attributes=service/name=your-service-name

### DISCUSSION THREAD

**Discussion by yangtaihsou** [open-telemetry/opentelemetry-java-instrumentation#9835] [Q&A]
Please help!
when exe:
 java -javaagent:opentelemetry-javaagent.jar -Dotel.resource.attributes=service.name=your-service-name -Dotel.traces.exporter=zipkin -jar target/java-agent-usage-0.0.1-SNAPSHOT.jar com.tracicng.oteldemo.javaagent.JavaAgentUsageApplication

Received a lot of error:

[otel.javaagent 2023-11-09 21:29:39:064 +0800] [main] INFO io.opentelemetry.javaagent.tooling.VersionLogger - opentelemetry-javaagent - version: 1.31.0
错误: 找不到或无法加载主类 .resource.attributes=service.name=your-service-name
原因: java.lang.ClassNotFoundException: /resource/attributes=service/name=your-service-name

**Comment by laurit** [comment]
you probably have a space between `-Dotel` and `.resource.attributes=...`

**Reply by yangtaihsou** [reply]
Problems caused by windows encryption software
